examples/node/package.json

```diff
@@ -31,6 +31,7 @@
   "devDependencies": {
     "@lens-protocol/eslint-config": "workspace:*",
     "@lens-protocol/prettier-config": "workspace:*",
+    "@swc/core": "^1.4.13",
```

Comment:
Is this needed by what other dep?

Reply:
this is supposed to make ts-node faster, this comes together with `"swc": true` in tsconfig file